RecoTauTag/RecoTau/python/RecoTauPiZeroProducer_cff.py

```diff
@@ -60,12 +59,12 @@
     ),
 )
 
-from RecoTauTag.RecoTau.ak4PFJetsLegacyTaNCPiZerosDefault_cfi import ak4PFJetsLegacyTaNCPiZerosDefault
-# ak4PFJetsLegacyTaNCPiZeros = ak4PFJetsLegacyHPSPiZerosDefault.clone(
-ak4PFJetsLegacyTaNCPiZeros = ak4PFJetsLegacyTaNCPiZerosDefault.clone(
+
+ak4PFJetsLegacyTaNCPiZeros = recoTauPiZeroProducer.clone(
     jetSrc = PFRecoTauPFJetInputs.inputJetCollection,
     minJetPt = PFRecoTauPFJetInputs.minJetPt,
     maxJetAbsEta = PFRecoTauPFJetInputs.maxJetAbsEta,
+    massHypothesis = cms.double(0.136),
```

Comment:
This could be left out as it is added by the fillDescription (also in the other configs above). Note that it is not explicitly contained in the first config 'ak4PFJetsLegacyHPSPiZeros'.

Reply:
Thank you @swozniewski : since we are cleaning, it is correct to clean everywhere.
Done with d4e42ee